Bottom: 1dbc02e1e8de01bc32291f3ba7db967008eafcda
Top:    b29f7201270828d53cadbe38613e33f4babfce6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 16:14:03 +0000

gts: Add cast to silence compiler warning


---

diff --git a/src/gts/edge.c b/src/gts/edge.c
index 47754de..fde7d87 100644
--- a/src/gts/edge.c
+++ b/src/gts/edge.c
@@ -100,7 +100,7 @@ void gts_edge_remove(GtsEdge *edge)
 {
   edge->segment.v1->segments = g_slist_remove(edge->segment.v1->segments, &edge->segment);
   edge->segment.v2->segments = g_slist_remove(edge->segment.v2->segments, &edge->segment);
-  edge_destroy(edge); 
+  edge_destroy(GTS_OBJECT (edge));
 }
 
 /**
